状态,  描述,                                     用例名,                文件名,                行号
状态,  描述,                                     用例名,                文件名,                行号
状态,  描述,                                     用例名,                文件名,                行号
Fail,用例通过,                                   c_case4,            c_case4.py,         line 34
